mir-module/src/main/resources/xsl/toc/solr-fields4toc.xsl

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Builds solr fields used for table of contents -->
+
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:xalan="http://xml.apache.org/xalan"
+  xmlns:mods="http://www.loc.gov/mods/v3" 
+  exclude-result-prefixes="xalan xlink mods">
+  
+  <xsl:import href="xslImport:solr-document:toc/solr-fields4toc.xsl" />
+
+  <xsl:template match="mycoreobject">
+    <xsl:apply-imports />
+    <xsl:apply-templates select="metadata/def.modsContainer/modsContainer/mods:mods/mods:relatedItem[contains('host series',@type)]" mode="toc" />
+  </xsl:template>
+  
+  <xsl:template match="mods:relatedItem[contains('host series',@type)]" mode="toc">
+    <xsl:apply-templates select="mods:relatedItem[contains('host series',@type)]" mode="toc" />
```

Comment:
For downward compatibility these apply-templates should only be called if toc-layout is defined.